Core/Core/Modules/ModuleList/ModuleListViewController.swift

```diff
@@ -126,6 +127,21 @@ public class ModuleListViewController: ScreenViewTrackableViewController, Colore
         tableView.tableFooterView?.setNeedsLayout()
         tableView.reloadData()
         scrollToModule()
+
+        if spinnerView.isHidden, emptyView.isHidden, errorView.isHidden {
+            setupBulkPublishButtonInNavBar()
+        }
+    }
+
+    private func setupBulkPublishButtonInNavBar() {
+        guard navigationItem.rightBarButtonItem == nil,
```

Comment:
This method is invoked from the update menu that gets called multiple times during loading. This nil check is just to avoid the menu setup being done unnecessarily again and again.